app/components/primer/beta/button.pcss

```diff
@@ -27,7 +27,7 @@
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  gap: var(--primer-control-medium-gap, 8px);
+  gap: var(--base-size-4, 4px);
```

Comment:
Is this a significant change for all our buttons?

Reply:
It actually hasn't been doing anything at all 🙈 I think it was left in by mistake. All buttons have the `content` layer inside that controls layout/gap. But we found a use case for it here where we render a label outside of that container I think? I'll double check that.